cmd/backup-manager/app/clean/clean.go

```diff
@@ -67,14 +68,14 @@ func (bo *Options) cleanRemoteBackupData(bucket string, opts []string) error {
 	destBucket := util.NormalizeBucketURI(bucket)
 	args := util.ConstructArgs(constants.RcloneConfigArg, opts, "deletefile", destBucket, "")
 	output, err := exec.Command("rclone", args...).CombinedOutput()
-	if err != nil {
-		if exitError, ok := err.(*exec.ExitError); ok {
-			if code := exitError.ExitCode(); code == 3 || code == 4 {
-				klog.Infof("cluster %s backup %s has already been deleted before", bo, bucket)
-				return nil
-			}
-		}
-		return fmt.Errorf("cluster %s, execute rclone deletefile command failed, output: %s, err: %v", bo, string(output), err)
+	if err != nil && !strings.Contains(string(output), "doesn't exist") {
+		return fmt.Errorf("cluster %s, execute rclone deletefile command to delete archive failed, output: %s, err: %v", bo, string(output), err)
+	}
+
+	args = util.ConstructArgs(constants.RcloneConfigArg, opts, "deletefile", fmt.Sprintf("%s.tmp", destBucket), "")
```

Comment:
when this `*.tmp` file will be created?

Reply:
After file is archived and uploaded to S3.
1. upload archive file to `*.tgz.tmp`
2. move `*.tgz.tmp` to `*.tgz`

Reply:
I see. Thanks!

Reply:
And this is for backup with `dumpling` only.